### PR TITLE
UNOMI-797 Fixes in migration documentation

### PR DESCRIPTION
**Pull request by sergehuber**
- Fixed anchor links
- Rewritten some parts to be more precise or easier to understand
- Added environment variable option for JSON schema debugging

### CODE REVIEW

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 1]
```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
...
+Since Apache Unomi is an engine, you've probably built multiple applications consuming its APIs, you might also have built extensions directly running in Unomi.
 
-As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<Enable debug mode>>. 
+As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<_enabling_debug_mode,enable debug mode>>.
 Doing so will display any errors when processing events (such as JSON Schema validations), and will provide useful indications towards solving issues.
 
 ==== Data Model changes
 
-There has been changes to Unomi Data model, please make sure to review those in the << what_s_new>> section of the user manual.
+There has been changes to Unomi Data model, please make sure to review those in the <<_whats_new_in_apache_unomi_2_0,What's new in Unomi 2>> section of the user manual.
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:32]: Did you verify the anchor was working fine ? (I'm asking specifically for the ' and . )

`<<_whats_new_in_apache_unomi_2_0,What's new in Unomi 2>>` linking to
```
=== What's new in Apache Unomi 2.0
```

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 2]
```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
...
+Apache Unomi ships, out of the box, with all of the necessary JSON Schemas for its own operation as well as all event types generated from the Apache Unomi Web Tracker but you will need to create schemas for any custom event you may be using.
 
-When creating your new schemas, reviewing debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console), 
-will point to errors in your schemas or will help you diagnose why the events are not being accepted.
+When creating your new schemas, you can setup debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console) that
+will point to errors in your schemas or will help you diagnose why the events are not being accepted. It is also possible to use the UNOMI_LOGS_JSONSCHEMA_LEVEL environment variable (by setting it to the `DEBUG` value) and then restarting Apache Unomi to accomplish the same thing. The second option is especially useful when using Docker containers. It is also possible to test if your events are valid with the a new API endpoint mapped at `/cxs/jsonSchema/validateEvent`.
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:42]: I find it a bit long to get all these in the same paragraph. You're trying to carry 3 pieces of info:
 - setup logs will help diagnose issue
 - ways to do that (3 options, karaf console, environment variable, api endpoint).

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 3]
```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
...
 ==== Elasticsearch version and capacity
 
-While still using Unomi 1.6, the first step will be to upgrade your Elasticsearch to 7.17.5. 
+While still using Unomi 1.6, the first step will be to upgrade your Elasticsearch to 7.17.5.
 Documentation is available on https://www.elastic.co/guide/en/elasticsearch/reference/7.17/setup-upgrade.html[Elasticsearch's website].
 
-Your Elasticsearch cluster must have enough capacity to handle the migration. 
-At a minimum, the required capacity must be greater than the size of the dataset in production + the size of the largest index.
+Your Elasticsearch cluster must have enough capacity to handle the migration.
+At a minimum, the required capacity storage capacity must be greater than the size of the dataset in production + the size of the largest index and any other settings should at least be as big as the source setup (preferably higher).
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:61]: I would use only + or only "and" as the reader is expected to calculate a sum

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc (hunk 1 above)
Comment (sergehuber) [reply manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:32]: I actually used IDEA's auto-completion for the anchor so yes I think it should be good.

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc (hunk 2 above)
Comment (sergehuber) [reply manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc]: Ok I've completely rewritten this paragraph and split it into multiple parts that should be simpler to understand.

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc (hunk 3 above)
Comment (sergehuber) [reply manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc]: I fixed this by breaking to another sentence.

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 4]
```diff
@@ -38,8 +38,19 @@ Once you updated your applications to align with Unomi 2 data model, the next st
 Any event (and more generally, any object) received through Unomi public endpoints do require a valid JSON schema.
 Apache Unomi ships, out of the box, with all of the necessary JSON Schemas for its own operation as well as all event types generated from the Apache Unomi Web Tracker but you will need to create schemas for any custom event you may be using.
 
-When creating your new schemas, you can setup debug messages in the logs (using: `log:set DEBUG org.apache.unomi.schema.impl.SchemaServiceImpl` in Karaf console) that
-will point to errors in your schemas or will help you diagnose why the events are not being accepted. It is also possible to use the UNOMI_LOGS_JSONSCHEMA_LEVEL environment variable (by setting it to the `DEBUG` value) and then restarting Apache Unomi to accomplish the same thing. The second option is especially useful when using Docker containers. It is also possible to test if your events are valid with the a new API endpoint mapped at `/cxs/jsonSchema/validateEvent`.
+When creating your new schemas, there are multiple ways of testing them:
+
+- Using a the event validation API endpoint mapped at : `/cxs/jsonSchema/validateEvent`
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:43]: ` Using the event validation API endpoint available at`

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 5]
```diff
@@ -38,8 +38,19 @@ Once you updated your applications to align with Unomi 2 data model, the next st
...
+- Using debug logs when sending events using the usual ways (using the `/context.json` or `/eventcollector` endpoints)
+
+Note that in both cases it helps to activate the debug logs.
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:46]: `...helps to activate the debug logs, you can activate them either:`

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc [hunk 6]
```diff
@@ -38,8 +38,19 @@ Once you updated your applications to align with Unomi 2 data model, the next st
...
+
+To active the debug logs you can activate them in two ways:
```
Comment (Fgerthoffert) [inline manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc:48]: remove (see above suggestion)

File: manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc (hunk 5 above)
Comment (sergehuber) [reply manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc]: Ok I'll change that

**Pull request merged**